fv3core/stencils/saturation_adjustment.py

```diff
@@ -1121,6 +1091,9 @@ def compute(
     tin = utils.make_storage_from_shape(peln.shape, utils.origin)
     q_cond = utils.make_storage_from_shape(peln.shape, utils.origin)
     qpz = utils.make_storage_from_shape(peln.shape, utils.origin)
+
+    do_qa = True
+
     satadjust_part1(
```

Comment:
can we just call this 'satadjust', since very fantastically there is only 1 part? woooo!

Reply:
Done!